cpp-client/deephaven/client/include/private/deephaven/client/server/server.h

```diff
@@ -67,6 +67,63 @@ struct ServerResponseHolder final : public CompletionQueueCallback {
   Response response_;
 };
 
+// Options for the creation of the underlying grpc channel.
+class ClientOptions {
+public:
+  typedef std::vector<std::pair<std::string, int>> int_options_t;
+  typedef std::vector<std::pair<std::string, std::string>> string_options_t;
+public:
```

Comment:
redundant `public:` ?

Reply:
I was used to doing that as a kind of "section separator" for typedefs.  I'll remove it.